.github/workflows/branch-reporter.yml

```diff
@@ -0,0 +1,31 @@
+name: Maven Build
+
+on:
+  pull_request:
+    types:
+      - opened
+      - review_requested
+      - edited
+      - synchronize
```

Comment:
@skayikci Is it possible to define `assigned `label in the types of pull request?

Here is the sample code shown below

```
pull_request:
  types:
    - assigned 
```

Reply:
done.